src/core/manager/mod.rs

```diff
@@ -1681,6 +1681,54 @@ mod tests {
     layout_trigger_test!(position_client; true; 10, Region::default(), true);
     layout_trigger_test!(layout_screen; true; 0);
 
+    #[test]
+    fn layout_trigger_test_follow_focus_cycle_client() {
+        let conn = RecordingXConn::init();
+        let conf = Config {
+            layouts: focus_test_layouts(true),
+            ..Default::default()
+        };
+        let mut wm = WindowManager::new(conf, conn, vec![], logging_error_handler());
+        wm.init().unwrap();
+        add_n_clients(&mut wm, 3, 0);
+        wm.focus_workspace(&Selector::Index(1)).unwrap();
+        add_n_clients(&mut wm, 3, 30);
+        wm.focus_workspace(&Selector::Index(0)).unwrap();
+        wm.conn.clear();
+        wm.cycle_client(Direction::Forward).unwrap();
+
+        let should_be_true = wm.conn.calls().iter().any(|c| c.0 == *"position_client");
```

Comment:
Can these assertions be rewritten without dereferencing the `&str` here? I'm sure it works but it's pretty funky...

I'd expect this to work?
```rust
assert!(wm.conn.calls().iter().any(|(method, _)| method == "position_client"))
```

Or alternatively, this sort of thing with the `RecordingXConn` is actually quite useful so it might be worthwile making it a method:
```rust
// in src/__test_helpers.rs
impl RecordingXConn {
    ...

    pub fn was_called(&self, method: &str) -> bool {
        self.calls().iter().any(|(m, _)| m == method)
    }

    pub fn was_not_called(&self, method: &str) -> bool {
        self.calls().iter().all(|(m, _)| m != method)
    }
}

// in your test
assert!(wm.conn.was_called("position_client"))
```